src/java_tools/junitrunner/javatests/com/google/testing/coverage/JacocoLCOVFormatterUninstrumentedTest.java

```diff
@@ -0,0 +1,157 @@
+
+  private StringWriter writer;
+  private IBundleCoverage mockBundle;
+
+  private static IClassCoverage mockIClassCoverage(String className, String packageName, String sourceFileName) {
+    IClassCoverage mocked = mock(IClassCoverage.class);
+    when(mocked.getName()).thenReturn(className);
+    when(mocked.getPackageName()).thenReturn(packageName);
+    when(mocked.getSourceFileName()).thenReturn(sourceFileName);
+    return mocked;
+  }
+
+  private Description createSuiteDescription(String name) {
+    Description suite = Description.createSuiteDescription(name);
+    suite.addChild(Description.createTestDescription(Object.class, "child"));
+    return suite;
+  }
+
+  @Before
+  public void setupTest() {
+    // Initialize writer for storing coverage report outputs
+    writer = new StringWriter();
+    // Initialize mock Jacoco bundle containing the mock coverage
+    // Classes
+    List<IClassCoverage> mockClassCoverages = Arrays.asList(
+      mockIClassCoverage("Foo", "com/example", "Foo.java")
+    );
+    // Package
+    IPackageCoverage mockPackageCoverage = mock(IPackageCoverage.class);
+    when(mockPackageCoverage.getClasses()).thenReturn(mockClassCoverages);
+    // Bundle
+    mockBundle = mock(IBundleCoverage.class);
+    when(mockBundle.getPackages()).thenReturn(Arrays.asList(mockPackageCoverage));
+  }
+
+  @Test
+  public void testVisitBundleWithSimpleUnixPath() throws IOException {
+    // Paths
+    ImmutableSet<String> execPaths = ImmutableSet.of(
+      "/parent/dir/com/example/Foo.java"
+    );
+    JacocoLCOVFormatter formatter = new JacocoLCOVFormatter(execPaths);
+    IReportVisitor visitor = formatter.createVisitor(new PrintWriter(writer), new TreeMap<String, BranchCoverageDetail>());
+    visitor.visitBundle(mockBundle, mock(ISourceFileLocator.class));
+    visitor.visitEnd();
+    String coverageOutput = writer.toString();
+    for (String sourcePath : execPaths) {
+      assertThat(coverageOutput).contains(sourcePath);
+    }
+  }
+
+  @Test
+  public void testVisitBundleWithSimpleWindowsPath() throws IOException {
+    // Paths
+    ImmutableSet<String> execPaths = ImmutableSet.of(
+      "C:/parent/dir/com/example/Foo.java"
+    );
+    JacocoLCOVFormatter formatter = new JacocoLCOVFormatter(execPaths);
+    IReportVisitor visitor = formatter.createVisitor(new PrintWriter(writer), new TreeMap<String, BranchCoverageDetail>());
+    visitor.visitBundle(mockBundle, mock(ISourceFileLocator.class));
+    visitor.visitEnd();
+    String coverageOutput = writer.toString();
+    for (String sourcePath : execPaths) {
+      assertThat(coverageOutput).contains(sourcePath);
+    }
+  }
+
+  @Test
+  public void testVisitBundleWithMappedUnixPath() throws IOException {
+    // Paths
+    String srcPath = "/some/other/dir/Foo.java";
+    ImmutableSet<String> execPaths = ImmutableSet.of(
+      String.format("%s%s%s", srcPath, JacocoLCOVFormatter.EXEC_PATH_DELIMITER, "/com/example/Foo.java")
```

Comment:
Style: `EXEC_PATH_DELIMITER` can be inlined here. The pattern is essentially API and it makes this test clearer to have a simple string.
This also means `JacocoLCOVFormatter.EXEC_PATH_DELIMTER` can be made `private`, not `public`.